### ISSUE TITLE
[Vue-ApexCharts] Toolbar doesn't work properly with xaxis.categories 

### ISSUE THREAD

**Issue by vystepanenko** [apexcharts/vue-apexcharts#9]
Hi!
Component: 
"vue": "^2.5.17",
"vue-apexcharts": "^1.1.1",
"apexcharts": "^1.5.1",

```
      options: {
        chart: {
          id: "idChart",
          toolbar: { show: true }
        },
        xaxis: { categories: ["1991", "1992", "1993", "1994", "1995", "1996"] }
      },
      series: [ { name: "series-1", data: [30, 40, 45, 50, 49, 60, 70, 91]  }]
```
In this case I have only download button. Even if specify tools options like this
`tools: {  download: true, selection: true, zoom: true, zoomin: true, zoomout: true, pan: true, reset: true },`

![image](https://user-images.githubusercontent.com/22811452/45541077-b88f7200-b816-11e8-93b3-59d8d8c8219e.png)
Zoom doesnt work in this case.

But without xaxis.categories I have all buttons and zoom works as expected
```
      options: {
        chart: {
          id: "idChart",
          toolbar: { show: true }
        },
      },
      series: [ { name: "series-1", data: [30, 40, 45, 50, 49, 60, 70, 91]  }]
```
![image](https://user-images.githubusercontent.com/22811452/45541268-44090300-b817-11e8-8487-4174dbfe8bbe.png)

**Comment by junedchhipa** [comment]
I will take a look soon, thanks for reporting

**Comment by junedchhipa** [comment]
Currently, Zooming and Panning are supported for the linear/DateTime x-axis.
So, you have to provide valid DateTime formats for zooming to work in categories

Make the following changes to your code and see how zooming is enabled
```javascript
var options = {
  chart: {
    type: 'bar'
  },
  series: [{
    name: 'sales',
    data: [30,40,45,50,49,60,70,91,125]
  }],
  xaxis: {
    type: 'datetime',
    categories: ['01/01/1991','01/01/1992','01/01/1993','01/01/1994','01/01/1995','01/01/1996','01/01/1997', '01/01/1998','01/01/1999']
  }
}

var chart = new ApexCharts(document.querySelector("#chart"), options);

chart.render();
```

[Demo](https://codepen.io/apexcharts/pen/KxBdBZ)

**Comment by vystepanenko** [comment]
With datetime zooming works perfectly. thnx

**Comment by jbrodriguez** [comment]
Hi, I'm looking into this library to incorporate in my app.

Is zooming for string categories in the roadmap ?

**Comment by vystepanenko** [comment]
It will be great if zoom will works on string categories too

**Comment by junedchhipa** [comment]
Related to https://github.com/apexcharts/apexcharts.js/issues/920

**Comment by junedchhipa** [comment]
Released v3.15.0 with the zooming enabled on category x-axis.
Example - https://apexcharts.com/javascript-chart-demos/column-charts/column-with-rotated-xaxis-labels/

Doc: https://apexcharts.com/docs/zooming-in-category-x-axis/
